controllers/NUgus_Controller/Makefile

```diff
@@ -65,6 +65,9 @@
 ###
 ###-----------------------------------------------------------------------------
 
+CFLAGS = -Wall
```

Comment:
```suggestion
CFLAGS = -Wall -Wpedantic -Wextra -O3 -fPIC
CXXFLAGS = ${CFLAGS} -std=c++11
```
The standard is required for r-value references.

Not sure why we're using a makefile, but if we're setting flags here, these ones are good

Reply:
Webots controllers is made with the provided makefile.

Reply:
Probably better that we use cmake, maintain consistency across all of our repos as much as possible

Reply:
Is it possible to include a `makefile.include` in cmake?

Reply:
No, that is against the way cmake works

Reply:
gist.github.com › gsc07
Webots Controller build using CMake · GitHub